src/styles/_header.scss

```
@@ -37,5 +37,10 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+    transition-duration: 300ms;
```

```suggestion
    transition-duration: color 300ms;
```
Завжди варто додавати властивість, для якої буде застосовано transition